package com.asapp.backend.challenge.domain.exceptions;

public class UserAlreadyExistsException extends Throwable {
}
